---
title: "R Notebook"
output: github_document
---


```{r}
library(dada2); packageVersion("dada2")
```

```{r}
metatable<-read.csv(file="Metatable.csv", sep=";")
```

```{r}
path <- "/home/rstudio/ADM_Final_Assessment/sequences"
list.files(path)

```

```{r}
fnFs <- sort(list.files(path, pattern="_1.fastq.gz", full.names = TRUE))
fnRs <- sort(list.files(path, pattern="_2.fastq.gz", full.names = TRUE))
```
```{r}
sample.names <- sapply(strsplit(basename(fnFs), "_"), `[`, 1)
```

```{r}
plotQualityProfile(fnFs[1:2])
```
```{r}
plotQualityProfile(fnRs[1:2])
```
#For the forward sequences we will trim the 30 last nucleotides (truncate the forward reads at position 270)
#For the Reverse sequences we will trim truncate at position 180

```{r}
filtFs <- file.path(path, "filtered", paste0(sample.names, "_F_filt.fastq.gz"))
filtRs <- file.path(path, "filtered", paste0(sample.names, "_R_filt.fastq.gz"))
names(filtFs) <- sample.names
names(filtRs) <- sample.names
```

```{r}
out <- filterAndTrim(fnFs, filtFs, fnRs, filtRs, truncLen=c(270,180),
              maxN=0, maxEE=c(2,2), truncQ=2, rm.phix=TRUE,
              compress=TRUE, multithread=FALSE)
head(out)
```

